packages/webr/R/shims.R

```diff
@@ -0,0 +1,8 @@
+#' shim base functions
+shim_webr <- function(){
```

Comment:
```suggestion
shim_install <- function(){
```

Do you mind if we name the function to match other setup functions in the webR package, e.g. `pager_install()`, `canvas_install()`?